.github/workflows/cypress.yml

```diff
@@ -0,0 +1,19 @@
+name: cypress
+on: [deployment_status]
+jobs:
+  e2e:
```

Comment:
Also use a verb (to be in sync with the other workflow configs in the repo)? For example: 'test' or 'run' or 'run-end-to-end-tests'?